Packs/Workday/Integrations/WorkdayIAMEventsGenerator/README.md

```diff
@@ -0,0 +1,78 @@
+Generates Workday dummy test reports and events. Used for testing/dev purposes.
+
+To use this integration for testing:
+1. Create an instance - Add a listening port and mark the *Long running instance* checkbox.
+2. Take the server with the listening port that has been configured in the integration settings,
+and configure the **Workday_IAM** instance with this url and invoke the *fetch_incidents*.
+
+For more information about how to configure the long running integration go to [Long Running HTTP Integrations](https://xsoar.pan.dev/docs/reference/articles/long-running-invoke).
+
+
+This integration uses mock reports (simulates the real reports that are coming from Workday).
+You can use the **generate-event** command to get all kinds of reports (hire, update, terminate and rehire)
```

Comment:
You can use the **generate-event** command to get all kinds of reports (hire, update, terminate, and rehire).